ComposableArchitecture.xcworkspace/contents.xcworkspacedata

```diff
@@ -7,9 +7,6 @@
    <FileRef
       location = "group:Examples/CaseStudies/CaseStudies.xcodeproj">
    </FileRef>
-   <FileRef
-      location = "group:Examples/LocationManager/LocationManager.xcodeproj">
-   </FileRef>
```

Comment:
We don't wanna remove this, can we bring it back?

Reply:
working on it now. Sorry about that